src/quacc/recipes/espresso/phonons.py

```diff
@@ -0,0 +1,142 @@
+"""Core recipes for espresso."""
+from __future__ import annotations
+
+from functools import partial
+from typing import TYPE_CHECKING
+
+from ase import Atoms
+
+from quacc import Job, flow, job
+from quacc.calculators.espresso.espresso import EspressoTemplate
+from quacc.calculators.espresso.utils import parse_ph_patterns
+from quacc.recipes.espresso._base import base_fn
+from quacc.recipes.espresso.core import static_job
+
+if TYPE_CHECKING:
+    from quacc.schemas._aliases.ase import RunSchema
+
+
+@job
+def phonon_job(
+    preset: str | None = None,
+    copy_files: str | list[str] | None = None,
+    parallel_info: dict[str] | None = None,
+    test_run: bool = False,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a basic ph.x calculation.
+
+    Parameters
+    ----------
+    preset
+        The name of a YAML file containing a list of parameters to use as
+        a "preset" for the calculator. quacc will automatically look in the
+        `ESPRESSO_PRESET_DIR` (default: quacc/calculators/espresso/presets).
+
```

Comment:
As a head's up, there can't be line breaks between the arguments otherwise it wouldn't render in the `mkdocs` documentation. A bit annoying, but it is what it is.

Also, please add `test_run` and any other kwargs to the docstring when you're done. Thanks!